manual/googlechrome-extensions/googlechrome-extension-template/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,18 @@
+$bits = Get-ProcessorBits
+$packageName = 'googlechrome-'
+$extensionID = ''
+
+if ($bits -eq 64)
+{
+  if (Test-Path -Path "HKLM:\SOFTWARE\Wow6432node\Google\Chrome\Extensions\$extensionID")
+  {
+    Write-Host "Extension already installed." -foreground "magenta" –backgroundcolor "blue"
+  } else {
+    New-Item -Path "HKLM:\SOFTWARE\Wow6432node\Google\Chrome\Extensions\$extensionID"
```

Comment:
tried adding `-Force` here, and that works.
This would then make it work if the user hasn't launched google chrome yet.